changelog.d/20250410_102306_8730430+m1yag1_sc_40144_add_run_manager_monitor_support.rst

```diff
@@ -0,0 +1,5 @@
+Added
+~~~~~
+
+- Added support to the create and update operations of the ``FlowsClient`` for
+  including ``run_managers`` and/or ``run_monitors`` principals. (:pr:`NUMBER`)
```

Comment:
We should refer to the methods by name. A client could have multiple operations which are 'create' or 'update' (for different resource types), and even when that's not the case, which of the following two is easier to resolve to some specific bit of code:
- "create operation of FlowsClient"
- "FlowsClient.create_flow"

I would probably go with
```suggestion
- ``FlowsClient.create_flow`` and ``FlowsClient.update_flow`` now support ``run_managers``
  and ``run_monitors``. (:pr:`NUMBER`)
```